pkg/operator/operatorclient.go

```diff
@@ -84,3 +84,12 @@ func (c OperatorClient) GetOperatorInstance() (*operatorv1.Storage, error) {
 	}
 	return instance, nil
 }
+
+func (c OperatorClient) GetOperatorStateWithQuorum(ctx context.Context) (*operatorv1.OperatorSpec, *operatorv1.OperatorStatus, string, error) {
+	instance, err := c.Client.Storages().Get(ctx, globalConfigName, metav1.GetOptions{})
+	if err != nil {
+		return nil, nil, "", err
+	}
+
+	return &instance.Spec.OperatorSpec, &instance.Status.OperatorStatus, instance.GetResourceVersion(), nil
+}
```

Comment:
What is this for? I can't find where it's called, maybe I'm missing something.

Reply:
Ah nevermind, [now I see it](https://github.com/openshift/vsphere-problem-detector/pull/152/files#diff-7aa77206e39ed184ec2081444dc5add93fe4b0ae88c8bce18303a394e4956244R175)